src/kubernetes-tool/templates/splash_screen.vue

```diff
@@ -33,15 +33,30 @@ limitations under the License.
                 <div class="input-container">
                     <label for="helmInput" class="hidden">{{ i18n.templates.splashScreen.helmTitle }}</label>
                     <i class="fas fa-dharmachakra"></i>
-                    <input id="helmInput"
-                           v-model="helmId"
-                           class="input"
-                           type="text"
-                           :placeholder="i18n.templates.splashScreen.helmTitle"
-                    />
-                    <button id="submitHelm" class="button is-primary" :click="submitHelm">
-                        {{ i18n.templates.splashScreen.submit }}
-                    </button>
+                    <vue-autosuggest
+                        v-model="helmId"
+                        :suggestions="helmSuggestions"
+                        :input-props="{id: 'helmInput', class: 'input', type: 'text', placeholder: i18n.templates.splashScreen.helmTitle, style: {width: '80%', textIndent: '30px'}, readonly: readonly}"
+                        :style="{width: '100%'}"
```

Comment:
Why not just make this a normal style attr?

Reply:
Because that makes a new CSS file.

Reply:
It doesn't, style elements make a new CSS file, style attributes stay on that element and are fine.